assets/parquet/config.yaml

```
@@ -0,0 +1,134 @@
+  # 1679490000
+  - '"json_extract_scalar(json_parse(line), ''$.c'') = ''WTWRTLOG'' AND CAST(json_extract_scalar(json_parse(line), ''$.attr.message.ts_sec'') AS bigint) > 1679490000;"'
+  # Query all JSON log lines where "ctx" is "FlowControlRefresher" and "attr.numTrimmed" equals 0
+  - '"json_extract_scalar(json_parse(line), ''$.ctx'') = ''FlowControlRefresher'' AND json_extract_scalar(json_parse(line), ''$.attr.numTrimmed'') = ''0'';"'
+  # - '"ctx: FlowControlRefresher AND attr.numTrimmed: 0"'
+queries_pairwise_arrays:
+  # Query all JSON log lines containing "attr" object with "tickets" field
+  - |
+    "contains(int_columns, 'attr.tickets')"
+  # Query all JSON log lines containing "id" field that equals to 22419
+  - |
+    "array_position(int_columns, 'id') > 0 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)